### PR TITLE
Add React to word list

### PR DESCRIPTION
**Pull request by josmperez**
Add entry for "React."

### CODE REVIEW

File: docs/sources/write/style-guide/word-list/index.md [hunk 1]
```diff
@@ -82,6 +82,9 @@ Use this rather than _hamburger menu_ or _kebab menu_.
 
 ## R
 
+### React
+
+Spell as React, not React.js or ReactJS.
```
Comment (clayton-cornell) [inline docs/sources/write/style-guide/word-list/index.md:87]: I'm curious... why not ReactJS? Thsi makes it clear you're referring to the JS library vs the ReactNative framework, and ReactJS is very commonly used within the JS dev community.

Comment (josmperez) [reply docs/sources/write/style-guide/word-list/index.md:87]: > I'm curious... why not ReactJS? This makes it clear you're referring to the JS library vs the ReactNative framework, and ReactJS is very commonly used within the JS dev community.

My reasoning is that react.dev, the official site, says the name is React; also Wikiipedia uses React. Other versions of the name exist, and hence there is the potential for inconsistencies.

Comment (josmperez) [reply docs/sources/write/style-guide/word-list/index.md:87]: Also I don't think anyone in Grafana uses React Native for anything.

**Review by clayton-cornell — APPROVED** [review]
LGTM

**Review by jdbaldry — APPROVED** [review]

**Pull request merged**